### PR TITLE
Ra cert fix for prod env

### PR DESCRIPTION
**Pull request by rhiananthony**
Please ensure the following when opening a PR:
- [x] Added or updated tests
- [x] Updated documentation
- [x] Applied style guidelines
- [x] Considered generalizability beyond our own use case

### CODE REVIEW

File: kubernetes/get_certs.sh [hunk 1]
```diff
@@ -9,8 +9,13 @@ else
     ENV="${LIRA_ENVIRONMENT}"
 fi
 
-SERVICE="lira"
-DOMAIN="pipelines.${ENV}.data.humancellatlas.org"
+if [ "${ENV}" == "prod" ]
```
Comment (rexwangcc) [inline kubernetes/get_certs.sh]: missing `;` at the end

**Review by rexwangcc — COMMENTED** [review]
One comment

File: kubernetes/get_certs.sh (hunk 1 above)
Comment (rhiananthony) [reply kubernetes/get_certs.sh]: Added :)

**Review by rexwangcc — APPROVED** [review]
👍

**Comment by rexwangcc** [comment]
retest this please

**Comment by rexwangcc** [comment]
retest this please

**Comment by rexwangcc** [comment]
@rhiananthony Mintegration test failed for a different reason :/

**Comment by rhiananthony** [comment]
We are merging this before the test passed because the test was broken

**Pull request merged**